docs/modules/configuration/pages/reporting.adoc

```diff
@@ -39,6 +39,122 @@ be used:
 report.translations.en.chart.duration.empty=No data to display
 ----
 
+4+^.^|[#_report_layout]_Report Layout_
+
+|`report.tabs.categories.enabled`
+a|`true` +
+`false`
+|`true`
+|Show or hide menu:Categories[] tab
+
+|`report.tabs.suites.enabled`
+a|`true` +
+`false`
+|`true`
+|Show or hide menu:Suites[] (aka menu:Batches[]) tab
```

Comment:
![image](https://github.com/user-attachments/assets/b1d3b0d9-498a-4fff-b0e9-3c2ef5d088d6)
Is this expected format of messages (I about '[]') ?
Shouldn't we add spaces after the colon?

Reply:
yes, it is, rendering is not correct without square brackets and no space is expected after colon: https://docs.asciidoctor.org/asciidoc/latest/macros/ui-macros/